owners/test/reviewer_selection.test.js

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2019 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const sinon = require('sinon');
+const {ReviewerSelection} = require('../src/reviewer_selection');
+const {OwnersRule, OwnersTree} = require('../src/owners');
+
+describe('reviewer selection', () => {
+  const sandbox = sinon.createSandbox();
+  const ownersTree = new OwnersTree();
+
+  const rootDirRule = new OwnersRule('OWNERS.yaml', ['root']);
```

Comment:
Maybe give them better fake names? It's a little confusing to think of "root" as a github username... e.g., alice, bob, carol :)

Reply:
Renamed "root" to "rootOwner", leaving the rest since it makes it easier to read what users the tests are expecting ("child" is more meaningful than "bob" (no offense Bob))